http/src/main/resources/init-resources/startup.sh

```diff
@@ -64,6 +64,17 @@ export WELDER_MEM_LIMIT=$(welderMemLimit)
 export MEM_LIMIT=$(memLimit)
 export USE_GCE_STARTUP_SCRIPT=$(useGceStartupScript)
 
+# Overwrite old cert on restart
+SERVER_CRT=$(proxyServerCrt)
+SERVER_KEY=$(proxyServerKey)
+ROOT_CA=$(rootCaPem)
+
+## This helps when we need to rotate certs.
```

Comment:
Hm, do we want to do this on _every_ resume? It should be a one-time thing -- could we only do it if the runtime has old certs?

Reply:
it's a bit tricky to test if the cert needs to an update tho...I'll give it a try